<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>220</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>120</horstretch>
    <verstretch>78</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>251</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>232</width>
      <height>210</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>232</width>
      <height>130</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="widget1">
     <attribute name="title">
      <string>클립보드 번역</string>
     </attribute>
     <widget class="QPushButton" name="clipButton">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>5</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>번역 시작</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>31</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>클립보드 텍스트를 번역합니다.</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget2">
     <attribute name="title">
      <string>파일 번역</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>번역할 폴더, 파일을 여기다 드래그&amp;드롭</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="trans_button">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>5</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>폴더 경로 설정 없이 누르면 탐색기가 열립니다</string>
      </property>
      <property name="text">
       <string>번역 시작</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>50</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget3">
     <attribute name="title">
      <string>설정</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>5</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>번역엔진 설정</string>
      </property>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>쿠우트랜스</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ezTransXP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>파파고</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DeepL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>구글번역기</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="apiButton">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>API키 넣기</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>60</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>번역할 언어 설정</string>
      </property>
      <widget class="QComboBox" name="comboBox_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>76</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>일본어</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>영어</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>중국어</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>스페인어</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>20</y>
         <width>56</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>로</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>20</y>
         <width>56</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>를</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_7">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>76</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>한국어</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>7</x>
        <y>120</y>
        <width>211</width>
        <height>61</height>
       </rect>
      </property>
      <property name="title">
       <string>클립보드 번역 설정</string>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>40</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>불투명도 조절</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>폰트 설정</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>80</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>일본어를 한국어로만 번역할 수 있습니다</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
